book/commands/input.md

```diff
@@ -25,5 +25,5 @@ usage: |
 
 Get input from the user, and assign to a variable
 ```shell
-> let user-input = (input)
+> let user_input = (input)
```

Comment:
I believe the command md files are generated systematically, so changing this here will just get overwritten next time they're generated. So, this should probably be changed in the crates/nu-command/src/platform/input.rs file.

Reply:
I've undone the change to this line and opened pr nushell/nushell#6716

Reply:
thanks